internal/providers/virtualbox/virtualbox.c

```diff
@@ -147,6 +147,35 @@ static int get_prop(int fd, uint32_t client_id, const char *name, void **value,
 	}
 }
 
+static int del_prop(int fd, uint32_t client_id, const char *name) {
```

Comment:
Is this code based on anything? Is there a useful reference link we could add here?